tests/rptest/tests/wasm_failure_recovery_test.py

```diff
@@ -81,6 +81,7 @@ def verify_results(self):
         return materialized_result_set_compare
 
 
+"""
 class WasmMeshFailureRecoveryTest(WasmFailureRecoveryTest):
```

Comment:
I can not use @ignore for this class. And test function locates in another file.

Reply:
Then I think we should just delete the code. We try not to have large amounts of uncommented code. Its OK if its deleted because the source is still in git history.